app/aem/all/build.gradle.kts

```diff
@@ -0,0 +1,37 @@
+plugins {
+    id("com.cognifide.aem.package")
+    id("maven-publish")
+}
+
+apply(from = rootProject.file("app/common.gradle.kts"))
+apply(from = rootProject.file("app/aem/common.gradle.kts"))
+
+description = "APM (AEM Permission Management) - AEM All-In-One Package"
+
+aem {
+    tasks {
+        packageCompose {
+            nestPackageProject(":app:aem:actions.checks")
```

Comment:
I want to make this package as optional, so please remove it from the this package.

Reply:
done